multi-agent/lib/mas/elements/llms/google_genai/google_genai.py

```
@@ -111,29 +113,123 @@ def stream(
                 continue
-
-            token = _extract_text_content(chunk.content)
-            if token:
-                yield token
-
-        if aggregated:
-            if hasattr(aggregated, 'content') and isinstance(aggregated.content, list):
-                aggregated.content = _extract_text_content(aggregated.content)
-            yield LangChainConverter.from_lc_message(aggregated)
+            sigs = sig_queues[queue_idx]
+            for part, sig in zip(fc_parts, sigs):
```

Ruff (0.15.12)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)